docs/CONFIG.md

```
@@ -29,6 +30,7 @@
   - [playerctl](#playerctl---up)
   - [ram-usage](#ram-usage---up)
   - [spotify-tui](#spotify-tui---up)
+  - [spr](#spr--up)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)